src/Microsoft.ML.Data/Transforms/ValueMappingTransformer.cs

```diff
@@ -0,0 +1,995 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Data.IO;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Transforms;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+[assembly: LoadableClass(ValueMappingTransformer.Summary, typeof(IDataTransform), typeof(ValueMappingTransformer),
+    typeof(ValueMappingTransformer.Arguments), typeof(SignatureDataTransform),
+    ValueMappingTransformer.UserName, "ValueMapping", "ValueMappingTransformer", ValueMappingTransformer.ShortName, "TermLookup",
+    DocName = "transform/ValueMappingTransformer.md")]
+
+[assembly: LoadableClass(ValueMappingTransformer.Summary, typeof(IDataTransform), typeof(ValueMappingTransformer), null, typeof(SignatureLoadDataTransform),
+    "Value Mapping Transform", ValueMappingTransformer.LoaderSignature)]
```

Comment:
>LoaderSignature [](start = 55, length = 15)

can you use two signature here, and get rid of second LoadDataTransform attribute?

Reply:
the second LoadDataTransform is for TermLookup, note the ValueMappingTransformer.TermLookupLoaderSignature


---
In reply to: [243465689](https://github.com/dotnet/machinelearning/pull/1710#discussion_r243465689) [](ancestors = 243465689)

Reply:
Got it! That makes sense. I will update.

---
In reply to: [243505500](https://github.com/dotnet/machinelearning/pull/1710#discussion_r243505500) [](ancestors = 243505500,243503134,243465689)